lib/core/network/api/services/dao_service.dart

```diff
@@ -45,4 +48,36 @@ class DaoService {
       return Result.error(HyphaError.fromError(error));
     }
   }
+
+  Future<Result<DaoData?, HyphaError>> getDaoById({
+    required Network network,
+    required String daoId,
+  }) async {
+    final String query =
+        '{"query":"query getDaoById(\$docId: String!) { queryDao(filter: {docId: {eq: \$docId}}) {docId details_daoName_n settings { settings_daoTitle_s, settings_daoDescription_s, settings_logo_s, settings_daoUrl_s } } }", "variables": {"docId": "$daoId"} } ';
+
+    final url = _remoteConfigService.graphQLEndpoint(network: network);
+    try {
+      final res = await _networkingManager.post(url, data: query);
```

Comment:
feels so weird to do `posts` in order to get data

Reply:
Well the body contains a rather elaborate query so... yeah. POST it is.